stock_landed_costs_mrp_subcontracting/__init__.py

```diff
@@ -0,0 +1,37 @@
+from . import models
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    # Before this module you receive partials from subcontracted vendor
+    # Backorders got wrong related origin_move_ids from manufacturing orders
+    # This script tries to correct associate
+    # stock moves on purchase_orders on field move_orig_ids
+    purchase_orders = env["purchase.order"].search([])
```

Comment:
Do you really need to iterate per purchase ?

As to avoid too much searches, you can do :

```suggestion
    moves = env["purchase.order"].search([("purchase_line_id", "!=", False) (...)])
```